pyproject.toml

```diff
@@ -13,6 +13,12 @@ dependencies = [
     "matplotlib~=3.0",
 ]
 
+[project.urls]
+Homepage = "https://github.com/PHOTOX/promdens"
+Documentation = "https://github.com/PHOTOX/promdens/blob/main/README.md"
+Repository = "https://github.com/PHOTOX/promdens/tree/main"
```

Comment:
Should this just be the same as Homepage?

Reply:
I changed Repository to `https://github.com/PHOTOX/promdens.git` according to the example on https://packaging.python.org/en/latest/guides/writing-pyproject-toml/. It is the same as Homepage since we don't have a separate web page for the code such as other programs. I'm quite happy if both the links lead here.